Sources_v3/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -83,18 +77,18 @@ class ChannelController_Tests: StressTestCase {
         XCTAssertEqual(controller.state, .remoteDataFetchFailed(ClientError(with: error)))
     }
     
-    func test_noChangesAreReported_beforeCallingStartUpdating() throws {
+    func test_changesAreReported_beforeCallingsynchronize() throws {
         // Save a new channel to DB
         client.databaseContainer.write { session in
             try session.saveChannel(payload: self.dummyPayload(with: self.channelId), query: nil)
         }
         
         // Assert the channel is not loaded
-        AssertAsync.staysTrue(controller.channel == nil)
-        AssertAsync.staysTrue(controller.messages.isEmpty)
+        AssertAsync.willBeTrue(controller.channel != nil)
+        AssertAsync.willBeTrue(!controller.messages.isEmpty)
     }
     
-    func test_startUpdating_fetchesExistingChannel() throws {
+    func test_synchronize_fetchesExistingChannel() throws {
```

Comment:
This should now be the default state, no?

Reply:
Discussed.